pagopa/pn-frontend
# replaced images

## Short description
Replaced images with brand

## List of changes proposed in this pull request
- Replaced images in static/images folder

## How to test
Landing -> go to "Cittadini" and "Enti" pages and check that images haven't brand
